mod-credits/src/main/java/gg/solarmc/loader/credits/CreditsConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ *  * dataloader
+ *  * Copyright © $DateInfo.year SolarMC Developers
```

Comment:
$DateInfo.year?

Reply:
I fucked it up, i have no clue how to set up the year replacement in intellij